app/src/main/java/com/rage/clamber/Adapters/WallsPageRecyclerViewAdapter.java

```diff
@@ -16,18 +19,18 @@
  */
 public class WallsPageRecyclerViewAdapter extends RecyclerView.Adapter<WallsPageRecyclerViewAdapter.WallsViewHolder>{
 
-    protected String[] walls;
+    protected ArrayList<WallSection> wallSections;
```

Comment:
So I updated the instance variable but if I try to change it in the constructor everything crashes. I am not sure why? It worked when I updated it in the Climbs Adapter?